docs/documentation_routes.js

```diff
@@ -77,6 +77,10 @@ router.get('/make-first-prototype/add-questions', function (req, res) {
   res.redirect('/docs/make-first-prototype/use-components')
 })
 
+router.get('/docs/views/templates/check-your-answers.html', function (req, res) {
+  res.redirect('/docs/views/templates/check-answers.html')
+})
+
```

Comment:
@joelanman Not sure I've done this ^ correctly...

Reply:
nearly! If you look at the other examples, they don't have `.html` as that doesnt appear in the URL

Reply:
Thanks @joelanman! Will fix